integration_tests/conftest.py

```diff
@@ -74,3 +80,189 @@ def pipeline_columns(sim, populations) -> List[str]:
         if column.startswith(pipeline)
     ]
     return pipeline_columns
+
+
+class FuzzyChecker:
+    """
+    This class manages "fuzzy" checks -- that is, checks of values that are
+    subject to stochastic variation.
+    It uses statistical hypothesis testing to determine whether the observed
+    value in the simulation is extreme enough to reject the null hypothesis that
+    the simulation is behaving correctly (according to a supplied verification
+    or validation target).
+
+    More detail about the statistics used here can be found at:
+    https://vivarium-research.readthedocs.io/en/latest/model_design/vivarium_features/automated_v_and_v/index.html#fuzzy-checking
+    """
+
+    def __init__(self) -> None:
+        self.comparisons_made = []
+
+    def fuzzy_assert_proportion(
+        self,
+        name: str,
+        boolean_values: pd.Series,
+        target_value: Optional[float] = None,
```

Comment:
Could use a tuple parameter here to avoid the complexity of the checks/optionals.